### ISSUE TITLE
On posts, the feature image width 100% seems to be overridden

### ISSUE THREAD

**Issue by annegentle** [mmistakes/so-simple-theme#304]
## Description
 
First of all, love your work. Using a couple of your themes in my projects, thanks!

Not sure when I noticed this, but it seems that the img width 100% is getting overridden. Screenshot of the Dev Console: 

<img width="212" alt="screen shot 2018-07-28 at 7 59 54 pm" src="https://user-images.githubusercontent.com/501981/43361822-f6566120-92a0-11e8-89f9-50f91c5fab0b.png">

I'm not great at debugging CSS but when I set width 100% the image spans the screen on desktop. 

<img width="201" alt="screen shot 2018-07-28 at 8 01 49 pm" src="https://user-images.githubusercontent.com/501981/43361827-1b3c056c-92a1-11e8-90b2-7b75e5153595.png">

## Environment

- So Simple version: jekyll-theme-so-simple 3.1.1
- Ruby gem or remote theme version: 
- Jekyll version: 3.8.3
- Git repository URL: https://github.com/justwriteclick/docslikecode
- Operating system: MacOS 10.13.6
- GitHub Pages hosted (if yes provide URL to site): Netlify, https://docslikecode.com

---

## Expected behavior

I like the image spanning the entire window, as shown below in the screenshot. Could you take a look? 

### Current look:
<img width="1249" alt="screen shot 2018-07-28 at 8 03 00 pm" src="https://user-images.githubusercontent.com/501981/43361837-5499a5a8-92a1-11e8-8d18-d6e86ca8b359.png">

### Desired outcome:

<img width="1498" alt="screen shot 2018-07-28 at 8 03 55 pm" src="https://user-images.githubusercontent.com/501981/43361842-6c7ea8f8-92a1-11e8-8e65-51b6ffe7ec83.png">

**Comment by mmistakes** [comment]
Yup, looks like setting `width: 100%` to those images will scale them up. Potential downside of doing this would be that it will emphasize artifacts due to smaller (lower quality) images being enlarged.

But the following will do the trick if you want to keep it isolated to just cover images.

```scss
.page-image {
  img {
    width: 100%;
  }
}
```

**Comment by mmistakes** [comment]
Are you sure you're using the latest version of the theme and not overwriting any of the Sass (CSS)?

I just checked the theme's demo site and feature images are correctly getting `width: 100%`. When I checked "docs like code" site I'm not seeing that.

![screen shot 2018-07-28 at 11 29 21 pm](https://user-images.githubusercontent.com/1376749/43362662-1aaf19dc-92be-11e8-8139-337c143f9b09.png)

Looking at the DLC source I see this, which would imply it's using an much older version that likely doesn't have the feature image fix.

```
<!--
  So Simple Jekyll Theme 3.0.1
  Copyright 2013-2018 Michael Rose - mademistakes.com \| @mmistakes
  Free for personal and commercial use under the MIT license
  https://github.com/mmistakes/so-simple-theme/blob/master/LICENSE
-->
```

**Comment by mmistakes** [comment]
[`Gemfile.lock`](https://github.com/justwriteclick/docslikecode/blob/build/Gemfile.lock#L56) is showing the older theme gem too. Run `bundle update` and then verify it's at 3.1.1 and the problem should be fixed for you.
